src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeffUnitTest.java

```diff
@@ -0,0 +1,218 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
```

Comment:
i think it would be cleaner to keep all in one package because we may move code around from allele specific to regular etc. but i'm fine with this too